docs/triggertemplates.md

```diff
@@ -78,8 +78,7 @@ Keep the following in mind:
   * `tekton.dev/triggers-eventid`:`<EventID>` to track resources created by a specific event.
 
 * To support arbitrary resource types, Tekton resolves resource templates internally as byte blobs. Because of this, Tekton only validates these
-  resources when processing an event rather than at the creation of the `TriggerTemplate`. Thus, you can **only** specify Tekton resources in a
-  `TriggerTemplate`.
+  resources when processing an event rather than at the creation of the `TriggerTemplate`.
```

Comment:
I think its worth noting here that the default EL [cluster role](https://github.com/tektoncd/triggers/blob/main/config/200-clusterrole.yaml#L86-L108)  only allows creating Tekton resources by default. 
Users will need to add additional roles as necessary to the EventListener's SA in order to be able to create non Tekton resources.